packages/kamino-sdk/src/Kamino.ts

```diff
@@ -2424,6 +2425,25 @@ export class Kamino {
     await this.openPosition(strategy, newPosition, priceLower, priceUpper, new Rebalancing()),
   ];
 
+  /**
+   * Get a list of rebalancing params
+   * @param rebalanceType the RebalanceTypeKind of the strategy
+   * @param rebalanceParams the raw rebalancing params
+   * @returns list of decoded rebalancing params
+   */
+
+  readPercentageRebalanceParams = (rebalanceType: RebalanceTypeKind, rebalanceParams: RebalanceRaw): Decimal[] => {
+    if (rebalanceType.kind == RebalanceType.Manual.kind) {
+      return [new Decimal(rebalanceParams.params[0]), new Decimal(rebalanceParams.params[1])];
+    } else if (rebalanceType.kind == RebalanceType.PricePercentage.kind) {
+      let lowerRangePercentage = rebalanceParams.params[0] + rebalanceParams.params[1] * 256;
```

Comment:
I think it might be safer to convert all these numbers or BN to Decimal and do something like:

```js
let lowerRangePercentage = new Decimal(rebalanceParams.params[0]).plus(new Decimal(rebalanceParams.params[1].mul(256));
```

Reply:
addressing it in a follow up PR

Reply:
thanks!